File: algorithms/sort/js/bubble.js
const arr = [3, 2, 1, 5, 4];

function sol(arr) {
  for (let i = 0; i < arr.length - 1; i++) {
    let swap = false;
    for (let j = 0; j < arr.length - i - 1; j++) {
      if (arr[j] > arr[j + 1]) {
        [arr[j], arr[j + 1]] = [arr[j + 1], arr[j]];
        swap = true;
      }
    }
  }
  return arr;
}

console.log(sol(arr));

File: algorithms/sort/py/bubble.py
# def sol(list):
#     for i in

list = [3, 2, 1, 5, 4]

for i in range(len(list) - 1):
    for j in range(len(list) - i - 1):
        if list[j] > list[j + 1]:
            temp = list[j]
            list[j] = list[j + 1]
            list[j + 1] = temp


print(list)
